CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 2.2.1
+
+* Update reflectable to enable the use of analyzer version 0.39.*. Adjust
```

Comment:
This description is largely irrelevant to the end-user, and covers three parts in one.

Consider starting with the bug fixed, using three bullet-points and limiting the descriptions of the lints
```
* Fix a bug concerned with the ordering of named
  parameters in a constructor declaration.
* Now compatible with `package:analyzer` ^ 0.39.0.
* Generated code is now  lint-free with package:pedantic x.y.z.
* package:reflectable itself  is lint-free with package:pedantic x.y.z.
```

Reply:
That's much better, thanks!